instrumentation/aws-lambda/aws-lambda-core-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdacore/v1_0/AwsLambdaInstrumentationModule.java

```diff
@@ -34,6 +35,8 @@ public boolean isHelperClass(String className) {
 
   @Override
   public List<TypeInstrumentation> typeInstrumentations() {
-    return singletonList(new AwsLambdaRequestHandlerInstrumentation());
+    return Arrays.asList(
```

Comment:
usually we static import the `asList` method when it is used in instrumentation module.

Reply:
some day I'm going to write an automation for this...